spec/octokit/client/organizations_spec.rb

```diff
@@ -332,22 +331,10 @@
       stub_get github_url("orgs/#{test_github_org}/memberships/#{test_github_login}")
       @client.organization_membership(
         test_github_org,
-        user: test_github_login,
-        accept: 'application/vnd.github.moondragon+json'
+        user: test_github_login
       )
       assert_requested :get, github_url("/orgs/#{test_github_org}/memberships/#{test_github_login}")
     end
-
-    it 'returns an organization membership for a given user by the orgs id' do
```

Comment:
This is no longer a valid test due to the REST API (OpenAPI descriptions) no longer allow ids to be passed in the query string for orgs